root/static/scripts/edit/MB/edit.js

```diff
@@ -140,7 +150,9 @@ import request from '../../common/utility/request';
 
             data.attributes = _(ko.unwrap(relationship.attributes))
                 .invokeMap('toJS')
-                .sortBy(function (a) { return a.type.id })
+                .sortBy(function (a) {
+                    return a.type.id;
+                })
```

Comment:
I think the inline function was fine (perhaps we should enable `allowSingleLine` on the rule), but we could also just replace it with an arrow function for simplicity:

```JS
.sortBy(a => a.type.id)
```

So I'd go with that.